README.md

```diff
@@ -2458,6 +2458,179 @@ You can query multiple elements, for example you can add the section with the im
 ```
  <Debugging query="product=Photoshop&image=test" />
 ```
+# Get Credential 
+
+The Get Credential component allows you to easily generate credentials and access code samples directly from the developer website. Child component should only work if it's inside the parent GetCredential component 
```

Comment:
We may want to add information that this is using the developer console apis to generate the credentials?